docs/get-started/start-teku.md

```diff
@@ -125,6 +125,36 @@ Content-Length: 0
   </TabItem>
 </Tabs>
 
+## Exit Codes
+
+**Exit Code 1**
+
+This code indicates a scenario where Teku has exited with a fatal error, however restarting Teku without changes is a logical step in correcting it. This code represents the error is related to something external to Teku.
+
+Example: If the beacon chain controller has issues starting P2P services on a particular port, the system will exit with a fatal error code of 1. This could correct itself with a restart.
+
+Example 2: If there is an issue migrating the database in the migrate database command, the system will exit with a fatal error code of 1. This issue has the potential to correct itself with a restart.
+
+**Exit Code 2**
+
+This code indicates a scenario where Teku has exited with a fatal error, but will not be corrected by itself if Teku is restarted.
+
+It is important to note that most user configuration errors fall into this category.
+
+Example: Within the migrate database command, if an invalid database version has been specified, Teku will exit with a fatal error code of 2. This error will not correct itself after a restart unless changes have been made to the set up.
```

Comment:
Rewrite:
If you specify an invalid database version in the migrate database command, Teku will exit with a code 2 fatal error. To correct this, make the appropriate changes to the setup and then restart Teku.